src/pages/Search/searchPw.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import './searchPw.scss';
+
+export default function SearchPw() {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [btn, setBtn] = useState(false);
+
+  function handleIdInput(e) {
+    setId(e.target.value);
+    btnActivate();
+  }
+  function handlePwInput(e) {
+    setPw(e.target.value);
+    btnActivate();
+  }
+  function btnActivate() {
+    return id.includes('@') && pw.length > 1 ? setBtn(true) : setBtn(false);
```

Comment:
아래와 같이 조건의 불리안 값을 이용하면 setBtn 함수를 중복해서 사용하는 코드를 간결화 할 수 있겠네요! 
```suggestion
    return setBtn( id.includes('@') && pw.length > 1 );
```